src/main/java/calc/StringAddCalculator.java

```diff
@@ -0,0 +1,43 @@
+package calc;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class StringAddCalculator {
+
+    public static final String REGEX = ",|:";
+    public static final String PATTERN = "//(.)\n(.*)";
+
+    public int splitAndSum(String text) {
+        if (text == null || text.isBlank()) {
+            return 0;
+        }
+        return split(text);
+    }
+
+    private static int split(String text) {
+        String[] splitText = text.split(",|:");
+        Matcher m = Pattern.compile(PATTERN).matcher(text);
+        if (m.find()) {
+            String customDelimiter = m.group(1);
+            splitText = m.group(2).split(customDelimiter);
+        }
+        return sum(splitText);
+    }
+
+    private static int sum(String[] splitText) {
+        int result = 0;
+        for (String s : splitText) {
+            isNegative(s);
+            result +=Integer.parseInt(s);
```

Comment:
코드 포맷팅 확인해주시면 좋을 것 같습니다.
```suggestion
            result += Integer.parseInt(s);
```